labs-ios-starter/City.swift

```diff
@@ -11,13 +11,14 @@ import Foundation
 struct City : Codable {
```

Comment:
I like the use of the extended description on this commit. I haven't usually used them, but github shows them prominently while reviewing, and I like the extra information!